src/mpid/ch4/netmod/ofi/ofi_types.h

```diff
@@ -397,6 +350,23 @@ typedef struct {
     char kvsname[MPIDI_KVSAPPSTRLEN];
     char pname[MPI_MAX_PROCESSOR_NAME];
     int port_name_tag_mask[MPIR_MAX_CONTEXT_MASK];
+
+    /* Capability settings */
+#ifdef MPIDI_OFI_ENABLE_RUNTIME_CHECKS
```

Comment:
In my code block above, the input is "NULL", not a subset of the features.  So you get back all capabilities from the provider.  If you have specific requirements, such as `scalable_endpoints = 0`, you can look through them and find the one you like.